sheetpr.go

```diff
@@ -33,14 +33,42 @@ type (
 	Published bool
 	// FitToPage is a SheetPrOption
 	FitToPage bool
-	// TabColor is a SheetPrOption
-	TabColor string
+	// TabColorIndexed is a TabColor option, within SheetPrOption
+	TabColorIndexed int
+	// TabColorRGB is a TabColor option, within SheetPrOption
+	TabColorRGB string
+	// TabColorTheme is a TabColor option, within SheetPrOption
+	TabColorTheme int
+	// TabColorTint is a TabColor option, within SheetPrOption
+	TabColorTint float64
+	// Equivalent to TabColorRGB, alias added for backwards compatibility
+	TabColor = TabColorRGB
 	// AutoPageBreaks is a SheetPrOption
 	AutoPageBreaks bool
 	// OutlineSummaryBelow is an outlinePr, within SheetPr option
 	OutlineSummaryBelow bool
 )
 
+const (
+	TabColorThemeUnset  int = iota - 1
```

Comment:
Thanks for your PR. Sorry, after reconsidering and referencing [the specification of tab color](https://github.com/sc34wg4/OOXMLSchemas/blob/759b6d4cbd0a353eb414a69a314c60ff8d389733/Part1/OfficeOpenXML-XMLSchema-Strict/sml.xsd#L3473-L3479). I think we need to return 0 if the theme index or tint is unset, so return the enumeration `TabColorThemeLight1` when theme index is unset, and return `0` when tint is unset, the `TabColorThemeUnset`, `TabColorIndexedUnset` and `TabColorTintUnset` is no longer required.

Reply:
You mean, because they're unsigned ints?
This means that if someone gets the theme for a cell that has no style, they'll get "light 1" (which isn't true)?

Reply:
I think setting tint to 0.0 by default makes sense as that means there's no tint, but it's not true for theme or indexed

Reply:
For ref.:
- Indexed = 0 is black color (https://docs.microsoft.com/en-us/dotnet/api/documentformat.openxml.spreadsheet.indexedcolors?view=openxml-2.8.1)
- Theme = 0 is light1, whose value depends on the theme

Would it make more sense to return a nil pointer for situations where the value is optional?

Reply:
You're right, just return `0` in float64 data type when the tint is unset, and return `-1` for theme or index unset. So, rename the enumeration `TabColorThemeUnset` to `TabColorUnsetValue` as return value when theme and index unset, the `TabColorIndexedUnset` and `TabColorTintUnset` is no longer required.

Reply:
Updated PR